### DISCUSSION TITLE
afk's in channel even if i restart the bot

### DISCUSSION THREAD

**Discussion by 012s** [SudhanPlayz/Discord-MusicBot#1246] [Q&A]
hello i want someone to make the bot afk's in channel 24/7 even when restarting, i tried doing it in ready.js and its working but when i use play command the play is not working, i know there is 24/7 code in botconfig but that dosen't help because it dosen't join the channel automatically after restarting.

**Comment by iseeface** [comment]
I don't think it can, and never will. The bot will automatically end the connection with all voice channels if you restart the console
